etc/plf/setenv-customize.sh

```diff
@@ -64,8 +64,13 @@ fi
 
 # eXo Onlyoffice addon
 if ${DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_ENABLED}; then
+    if ${DEPLOYMENT_APACHE_HTTPS_ENABLED}; then
+        SCHEMA=https
```

Comment:
Would be better to use a more precise variable like `DEPLOYMENT_ONLYOFFICE_DOCUMENTSERVER_SCHEME` instead of a generic `SCHEMA` variable

Reply:
variable name updated